<launch>
  <arg name="config_file" default="$(find yac_ros)/config/euroc-calib_imucam.yaml" />

  <!-- Play ROS bag -->
  <node pkg="rosbag" type="play" name="record" output="screen" args="-r 0.3 -q /data/euroc/rosbags/imu_april.bag"/>

  <!-- &#60;&#33;&#45;&#45; RVIZ &#45;&#45;&#62; -->
  <!-- <node type="rviz" name="rviz" pkg="rviz" -->
  <!--   args="-d $(find yac_ros)/config/yac.rviz" -->
  <!--   required="true" -->
  <!--   output="screen" /> -->

  <!-- NBV -->
  <node pkg="yac_ros" type="calib_nbt_node" name="yac_node" required="true" output="screen">
    <param name="config_file" value="$(arg config_file)" />
    <param name="cam0_topic" value="/cam0/image_raw" />
    <param name="cam1_topic" value="/cam1/image_raw" />
    <param name="imu0_topic" value="/imu0" />
  </node>
</launch>
